<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Filter</class>
 <widget class="QWidget" name="Filter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>628</width>
    <height>723</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#Filter {
background-color: rgb(0, 0, 34);
}

QLabel#Instructions {
color: rgb(255, 255, 255);
font: 24pt &quot;FreeSans&quot;;
}

QLabel#Price {
color: rgb(255, 255, 255);
font: 16pt &quot;FreeSans&quot;;
}

QLabel#Date {
color: rgb(255, 255, 255);
font: 16pt &quot;FreeSans&quot;;
}

QLabel#Discount {
color: rgb(255, 255, 255);
font: 16pt &quot;FreeSans&quot;;
}

QLabel#Amount{
color: rgb(255, 255, 255);
font: 16pt &quot;FreeSans&quot;;
}

QLabel#City {
color: rgb(255, 255, 255);
font: 16pt &quot;FreeSans&quot;;
}

QLabel#Unit {
color: rgb(255, 255, 255);
font: 16pt &quot;FreeSans&quot;;
}

QLabel#Shop {
color: rgb(255, 255, 255);
font: 16pt &quot;FreeSans&quot;;
}

QLabel#Comment {
color: rgb(255, 255, 255);
font: 16pt &quot;FreeSans&quot;;
}

QLabel#Product {
color: rgb(255, 255, 255);
font: 16pt &quot;FreeSans&quot;;
}

QPushButton#Button_Search{
color: rgb(255, 255, 255);
border-color: rgb(255, 255, 255);
border: 2px solid rgb(255, 255, 255);
border-radius: 20px;
font: 12pt &quot;FreeSans&quot;;
}

QPushButton#Button_Back{
color: rgb(255, 255, 255);
border-color: rgb(255, 255, 255);
border: 2px solid rgb(255, 255, 255);
border-radius: 20px;
font: 12pt &quot;FreeSans&quot;;
}

QPushButton:hover#Button_Search{
border: 3px solid rgb(255, 255, 255);
background-color: rgb(0, 0, 53);
}

QPushButton:hover#Button_Back{
border: 3px solid rgb(255, 255, 255);
background-color: rgb(0, 0, 53);
}

QLineEdit:hover#Product_choose,
QLineEdit:hover#Price_choose,
QLineEdit:hover#Date_choose,
QLineEdit:hover#Amount_choose,
QLineEdit:hover#Unit_choose,
QLineEdit:hover#Discount_choose,
QLineEdit:hover#Comment_choose,
QLineEdit:hover#City_choose,
QLineEdit:hover#Shop_choose{
border: 3px solid rgb(255, 255, 255);
}


QLineEdit#Product_choose,
QLineEdit#Price_choose,
QLineEdit#Date_choose,
QLineEdit#Amount_choose,
QLineEdit#Unit_choose,
QLineEdit#Discount_choose,
QLineEdit#Comment_choose,
QLineEdit#City_choose,
QLineEdit#Shop_choose
{
color: rgb(255, 255, 255);
font: 14pt &quot;FreeSans&quot;;
border: 2px solid rgb(255, 255, 255);
background-color: rgb(0, 0, 53);
border-radius: 10px;
padding-left: 20px;
padding-right: 20px;
}


QLabel#Message{
color: rgb(255, 0, 0);
font: 16pt &quot;FreeSans&quot;;
}</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>621</width>
     <height>701</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="Instructions">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>20</y>
      <width>451</width>
      <height>39</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>FreeSans</family>
      <pointsize>24</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Výběr filtru - porovnávat podle:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Product_choose">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>100</y>
      <width>341</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="Product">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>100</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Produkt:</string>
    </property>
   </widget>
   <widget class="QLabel" name="Message">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>550</y>
      <width>541</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_Search">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>600</y>
      <width>200</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>FreeSans</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Vyhledat</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_Back">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>600</y>
      <width>200</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>FreeSans</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Zavřít okno</string>
    </property>
   </widget>
   <widget class="QLabel" name="Price">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>150</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Cena:</string>
    </property>
   </widget>
   <widget class="QLabel" name="Date">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>200</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Datum (dd.mm.rrrr):</string>
    </property>
   </widget>
   <widget class="QLabel" name="Amount">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>250</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Množství:</string>
    </property>
   </widget>
   <widget class="QLabel" name="Unit">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>300</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Jednotka:</string>
    </property>
   </widget>
   <widget class="QLabel" name="Discount">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>350</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Sleva (ano/ne):</string>
    </property>
   </widget>
   <widget class="QLabel" name="Shop">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>400</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Obchod:</string>
    </property>
   </widget>
   <widget class="QLabel" name="City">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>450</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Město:</string>
    </property>
   </widget>
   <widget class="QLabel" name="Comment">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>500</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Komentář:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Price_choose">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>150</y>
      <width>341</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="Date_choose">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>200</y>
      <width>341</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="Amount_choose">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>250</y>
      <width>341</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="Unit_choose">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>300</y>
      <width>341</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="Discount_choose">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>350</y>
      <width>341</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="City_choose">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>450</y>
      <width>341</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="Comment_choose">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>500</y>
      <width>341</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="Shop_choose">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>400</y>
      <width>341</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
